ak/ak_build.py

```diff
@@ -225,6 +225,8 @@ def main(self, *args):
         config_file = self.config
         self._ensure_viable_installation(config_file)
         self._generate_links(config_file)
+        force_directory = get_repo_key_from_spec(self.directory) 
```

Comment:
if self.directory is False `ak build` failed

```
  File "/home/dav/dvp/py/ak/ak/ak_build.py", line 228, in main
    force_directory = get_repo_key_from_spec(self.directory) 
  File "/home/dav/dvp/py/ak/ak/ak_build.py", line 60, in get_repo_key_from_spec
    elif key[0:2] == './':
TypeError: 'NoneType' object is not subscriptable

```

It works for you ?

Reply:
What is the spec ?

Reply:
```
bank-statement-paypal:
    modules:
        - account_bank_statement_import_paypal
    src: https://github.com/akretion/bank-statement-import 8.0-paypal

```

Reply:
Proposed fix https://github.com/akretion/ak/pull/76